plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/SaveModelActionHandler.java

```diff
@@ -56,6 +61,7 @@ private void saveModelState(final GModelState modelState) {
             LOG.error(e);
          }
       });
+      modelSourceWatcher.continueWatching(modelState);
```

Comment:
Should we do that in a try-finally-block or do we assume that if there is an error when saving the model state we also want to stop watching the model source?

Reply:
Good idea, done!